test/token/ERC1155/ERC1155.behavior.js

```diff
@@ -208,7 +208,7 @@ function shouldBehaveLikeERC1155 ([minter, firstTokenHolder, secondTokenHolder,
       });
 
       it('reverts when transferring more than balance', async function () {
-        await expectRevert(
+        await expectRevert.unspecified(
```

Comment:
Should https://github.com/OpenZeppelin/openzeppelin-test-helpers/issues/149 be implemented, tests like these would not lose precision as much as in this current PR.